akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -976,6 +982,12 @@ trait LoggingAdapter {
    * @see [[LoggingAdapter]]
    */
   def warning(template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any): Unit = { if (isWarningEnabled) notifyWarning(format(template, arg1, arg2, arg3, arg4)) }
+  /**
+   * Message template with varargs replacement arguments.
+   * @see [[LoggingAdapter]]
+   */
+  @annotation.varargs
+  def warning(template: String, arg1: Any, arg2: Any, arg3: Any, arg4: Any, args: Any*): Unit = { if (isWarningEnabled) notifyWarning(format(template, Seq(arg1, arg2, arg3, arg4) ++ args:_*)) }
```

Comment:
please start this and others on a new line
(we should try to use < 120 char lines)